src/Sulu/Bundle/WebsiteBundle/Tests/Unit/Sulu/Bundle/WebsiteBundle/Routing/ContentRouteProviderTest.php

```diff
@@ -725,6 +725,50 @@ public function testGetCollectionTrailingSlash()
         $this->assertEquals('de/qwertz', $route->getDefaults()['url']);
     }
 
+    public function testGetCollectionTrailingSlashWithQueryParams()
+    {
+        $attributes = $this->prophesize('Sulu\Component\Webspace\Analyzer\Attributes\RequestAttributes');
```

Comment:
I'm using FQN here on purpose to avoid adding a `use` operator on the top of the page, and thus easing the merge.

Reply:
always use  the `::class` and import the class with `use` it helps later if you need to refractor code or find usages of a class by IDEs.

Reply:
Ok, done. I thought to impose as little changes as possible.